mcp/cloudrun_locals.tf

```diff
@@ -1,7 +1,20 @@
 //noinspection HILUnresolvedReference
 locals {
   cloudrun_default = {
-    annotations = {"autoscaling.knative.dev/maxScale" = "1000", "run.googleapis.com/client-name" = "terraform"}
+    metadata = {
+      annotations = {
+        "run.googleapis.com/client-name"    = "terraform"
+        "run.googleapis.com/ingress"        = "all"
```

Comment:
`internal` and `internal-and-cloud-load-balancing` are beta features so set default to `all`